docker/kubernetes/README.md

```diff
@@ -0,0 +1,260 @@
+# Running the Gameroom Demo in Kubernetes
+
+This procedure explains how to run the
+[Gameroom demo](https://github.com/Cargill/splinter/tree/master/examples/gameroom)
+with
+[Kubernetes](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/).
+This environment uses [Minikube](https://kubernetes.io/docs/setup/minikube/) to
+deploy two containerized Splinter nodes in a local Kubernetes cluster inside a
+virtual machine (VM) on your computer. This single pod deployment for each
+organization isn't intended to depict a production-like scenario. This is to
+demonstate the Kubernetes primitives.
+
+This procedure walks you through:
+
+* Generating keys and updating the key registry
+* Creating ConfigMaps for the key registry and node registry
+* Starting Gameroom
+* Creating users and logging in to the web app
+* Playing tic-tac-toe
+
+## Prerequisites
+
+You'll need Minikube installed and started and kubectl to complete this
+walkthrough. For installation instructions, see the
+[Minikube installation documentation](https://kubernetes.io/docs/tasks/tools/install-minikube/).
+
+## Deploy Gameroom
+
+### Step 1: Generate keys
+
+1. Run a Job to generate user keys.
+
+   `$ kubectl apply -f https://raw.githubusercontent.com/Cargill/splinter/master/docker/kubernetes/create-gameroom-keys.yaml`
+
+1. View the output from the Job.
+
+  ```
+  $ jobpod=$(kubectl get pods --selector=job-name=gameroom-keys --output=jsonpath='{.items[*].metadata.name}')
+  $ kubectl logs $jobpod
+  ```
+
+   The output will be similar to this example:
+
+  ```
+  alice.priv: 36a7e8cc7c81fdc7c7278aa157a732c283b9ba0e2ee4ec363992d3d96bad8207
+  alice.pub: 031bc8f3a326766ce628295ad6f01c49fd416306a0cf55cca42a6b87e3eccc3c8f
+  bob.priv: 8013752e26576e27e31105f6cf0f4537dc8a381a5b347a09959292cb4b17b388
+  bob.pub: 029d011c4c4f058b2b5a4cdff1dbe1749809cb121eb45dc766292a26d3a9b84fcd
+  ```
+
+1. Copy these values to a scratchpad or keep them available in your terminal,
```

Comment:
Tip: To get Markdown to number the last step as 3 (instead of 1), indent the code block by 4 spaces.  That tells Markdown that the code is part of step 2.